.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,38 +1,49 @@
-<!--- 
-Thanks for contributing to manim!
-**Please ensure that your pull request works with the latest version of manim from this repository.**
-You should also include:
-  1. The motivation for making this change (or link the relevant issues)
-  2. How you tested the new behavior (e.g. a minimal working example, before/after
-     screenshots, gifs, commands, etc.) This is rather informal at the moment, but
-     the goal is to show us how you know the pull request works as intended.
-If you don't need any of the optional sections, feel free to delete them to prevent clutter.
--->
-
-## List of Changes
-<!-- List out your changes one by one like this:
-- Change 1
-- Change 2
-- and so on..
+<!--
+Thank you for contributing to manim!
 
-Be sure to note your changes in the [changelog](docs/source/changelog.rst) if your
-changes warrant it!
+Please ensure that your pull request works with the latest
+version of manim from this repository.
 -->
 
 ## Motivation
-<!-- Why you feel your changes are required. -->
+<!-- Outline your motiviation: why do you feel that your
+changes are required? -->
```

Comment:
Things aren't really ever required in an absolute sense, only required in order to achieve something else. I think it'd be clearer to ask what problem the changes solve.

Reply:
That's fair. I'll change it to "Outline your motivation: what problem do your changes solve?" -- I was thinking about something like that originally, but didn't want to exclude PRs that, e.g., implement new features. But I guess that a missing feature can also be seen as a problem. 

Otherwise a wording like "in how far do your changes improve the library?" would also work; do you have any preference between these two directions?

Reply:
"In what way do your changes improve the library?" SGTM

Reply:
Done: 6f3315cd